docker/RELEASING.md

```diff
@@ -0,0 +1,53 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Bookkeeper release guide
+
+## Docker
+
+1. Check the latest GitHub release version
+
+    ```bash
+    export BK_RELEASE_VERSION=`curl -s https://api.github.com/repos/apache/bookkeeper/releases/latest | jq -r '.tag_name' | sed "s/release-\([0-9.]*\).*/\1/"`
```

Comment:
Use `BK_RELEASE_VERSION=curl -s https://api.github.com/repos/apache/bookkeeper/tags | jq -r '.[0].name' | sed 's/release-\([0-9.]*\).*/\1/'` to get `BK_RELEASE_VERSION`? 
If we release BK 4.14.6 after the BK 4.15.3 release, we will get `BK_RELEASE_VERSION` as 4.14.6 instead of 4.15.3?